ravelin.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://developer.ravelin.com'
   spec.license       = 'MIT'
 
-  spec.add_dependency('faraday', '~> 0.9')
+  spec.add_dependency('faraday', '~> 0.15.4')
```

Comment:
You'd normally just do a `~>` with the first 1 or 2 components, e.g. `~> 0.15`. If you want it to be at least 0.15.4 then you'd do `'~> 0.15', '>= 0.15.4'`, but I don't _think_ it should matter here?